tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -1641,6 +1641,28 @@ bool SpirvEmitter::validateVKAttributes(const NamedDecl *decl) {
     }
   }
 
+  // a VarDecl should have only one of vk::ext_builtin_input or
```

Comment:
Shall this case be handled?

```hlsl
cbuffer Constants {
    [[vk::ext_builtin_output(/* FragStencilRefEXT */ 5014)]]
    int builtin;
}
```

Right now it works, and builtin becomes constant (since in a cbuffer) so cannot be assigned.

Reply:
I think the error message for assigning to a constant is sufficient; I don't think there needs to be a special error message for declaring an output as constant.